.mvn/modernizer/violations.xml

```diff
@@ -158,6 +158,12 @@
         <comment>Prefer ConfigurationInstantiator.newEmptyConfiguration() for two reasons: (1) loading default resources is unlikely desired and (2) ConfigurationInstantiator adds additional safety checks</comment>
     </violation>
 
+    <violation>
+        <name>org/apache/hadoop/fs/FileSystem.close:()V</name>
```

Comment:
Will this be caught when the code uses explicit type, eg. SyncingFileSystem?
i'd guess the code might be compiled with actual implementation method name.

Maybe the shared FileSystem instances should be wrapped with something that prevents inadvertent close?

Reply:
Yes, as long as it's not cast to `Closeable`. We're getting rid of `FileSystem` usage so I'd rather not make an invasive change like that.